pkg/bootstrapper/kubelet.go

```diff
@@ -0,0 +1,255 @@
+package bootstrapper
+
+import (
+	"fmt"
+	"time"
+
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+// kubeletService struct contains the kubelet specific service information
+type kubeletService struct {
+	// obj is a pointer to the Windows service object
+	obj *mgr.Service
```

Comment:
I thought you wouldn't need this field, you could have directly embedded the mgr.Service and that would have given the necessary options?

Reply:
that is right, but I saw no harm in keeping this field to access the service object as I had already implemented using it.